src/military/military.cpp

```diff
@@ -5202,6 +5210,8 @@ void update_land_battles(sys::state& state) {
 				str_damage = std::min(str_damage, cstr);
 				state.world.regiment_get_pending_damage(att_front[i]) += str_damage;
 				cstr -= str_damage;
+				attacker_casualties += str_damage;
```

Comment:
you seem to be missing the equivalent addition for the backline inflicted defender casualties (there should be 2 of each, and you have only one place defender casualties are increases)

Reply:
isn't already in line 5176?

Reply:
I see two attacker casualties += but only one defender casualties +=; there should be two of each (but I was just guessing about which was was dropped since I am just looking at the changes without much context here)

Reply:
well, idk if this is what you meant but i noticed that the front line damage dealt was flipped (so the attacker fronline damage dealt was added to attacker_casualties instead of defender_casualties)